contrib/pg_tde/t/expected/001_basic.out

```diff
@@ -1,16 +1,43 @@
 CREATE EXTENSION IF NOT EXISTS pg_tde;
 SELECT extname, extversion FROM pg_extension WHERE extname = 'pg_tde';
-pg_tde|1.0-rc
+ extname | extversion 
+---------+------------
+ pg_tde  | 1.0-rc
+(1 row)
+
+CREATE TABLE test_enc(id SERIAL,k INTEGER,PRIMARY KEY (id)) USING tde_heap;
+psql:<stdin>:1: ERROR:  principal key not configured
```

Comment:
It would be nice to also show the error codes here, that would give us one more reminder to fix them.

Reply:
Maybe, the issue is that it is not in our SQL tests where I imagine most of these errors will be. Hmmm.